README.md

```diff
@@ -1 +1,273 @@
-# Hello-Word
\ No newline at end of file
+# Hello Mcroverse
```

Comment:
- [ ] Kindly ensure thst you fill out all the sections of the readme by adding descriptive texts that better describes your project. This is what the previous reviewer meant by personalizing your readme. Currently, most sections of your readme have not been properly edited with relevsnt contents. Kindly add contents to all the sections. 

Please note that only FAQ section is optional.